lib/models/note.js

```diff
@@ -584,5 +586,34 @@ module.exports = function (sequelize, DataTypes) {
     return operations
   }
 
+  Note.lockOldGuestNotes = async function (seconds, permission) {
+    if (typeof seconds !== 'number') {
+      throw new Error("Parameter 'seconds' has to be of type 'number'")
+    }
+    if (!permissionTypes.includes(permission)) {
+      throw new Error('Unexpected permission for locking old guest notes')
+    }
+    const notes = await Note.findAll({
+      attributes: ['id', 'ownerId', 'permission', 'createdAt'],
```

Comment:
Do we need the extra fields in the results, because we want to filter by them or could they be omitted?

Reply:
Originally I added those since they are all used in the request, but dropped it down. Main reason to specify attributes in first place, so the database doesn't pull all content each time we try to expire the notes.